api/jobs.yaml

```diff
@@ -116,6 +131,40 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    properties:
+      isRequired:
+        type: boolean
+        description: Whether or not authentication is required
+      scopes:
+        type: array
+        description: OAuth 2.0 requested scopes
+        items:
+          type: string
+  CapabilitiesResponse:
+    description: Specification of the backend's capabilities
+    properties:
+      authentication:
+        $ref: '#/definitions/AuthenticationCapability'
+      displayFields:
+        type: object
+        description: >
+          Fields on QueryJobsResult returned from POST /jobs/query
+          populated on some or all jobs. The fields are mapped to their
+          display names, in order of importance. Extended fields and
+          labels can be included, such as 'label.foo' or
+          'extendedFields.userId'
+      commonLabels:
+        type: array
+        description: Common labels which are present on most jobs returned
+        items:
+          type: string
+      extendedQueryFields:
+        type: array
+        description: Fields on ExtendedQueryFields which are queryable
```

Comment:
Should this say "Fields on QueryJobsResult which are queryable"?

Reply:
This really only applies to fields on the [ExtendedQueryFields](https://github.com/DataBiosphere/job-manager/pull/208/files#diff-4945231733bba41c658978333f49c3dfR273) model. It is a bit confusing submitting this before that model has been added but I think it should be fine in the short term.

Reply:
Oh, I see! Thanks for the clarification